draft-wussler-openpgp-pqc.md

```diff
@@ -1150,129 +1073,128 @@ Algorithm ID reference | Curve           | Field size | Public key | Secret key
 40                     | brainpoolP384r1 | 48         | 97         | 48         | 48                | 48
 
-### Dilithium signatures {#dilithium-signature}
+### ML-DSA signatures {#mldsa-signature}
 
-The procedure for Dilithium signature generation is the function `Sign(sk, M)`
-given in Figure 4 in [DILITHIUM-Subm], where `sk` is the Dilithium private key
-and `M` is the data to be signed. OpenPGP does not use the optional randomized
-signing given as a variant in the definition of this function, i.e. `rho' :=
-H(K || mu)` is used. The signing function returns the Dilithium signature. That
-is, to sign with Dilithium the following operation is defined:
+For ML-DSA signature generation the default hedged version of `ML-DSA.Sign`
+given in [FIPS-204] is used. That is, to sign with ML-DSA the following
+operation is defined:
 
-    (dilithiumSignature) <- dilithium.sign(dilithiumPrivateKey,
-                                           dataDigest)
+    (mldsaSignature) <- ML-DSA.Sign(mldsaPrivateKey, dataDigest)
 
-The procedure for Dilithium signature verification is the function `Verify(pk,
-M, sigma)` given in Figure 4 in [DILITHIUM-Subm], where `pk` is the Dilithium
-public key, `M` is the data to be signed and `sigma` is the Dilithium
-signature. That is, to verify with Dilithium the following operation is
+For ML-DSA signature verification the algorithm ML-DSA.Verify given in
+[FIPS-204] is used.  That is, to verify with ML-DSA the following operation is
 defined:
 
-    (verified) <- dilithium.verify(dilithiumPublicKey, dataDigest,
-                                   dilithiumSignature)
+    (verified) <- ML-DSA.Verify(mldsaPublicKey, dataDigest, mldsaSignature)
 
-Dilithium has the parameterization with the corresponding artifact lengths in
-octets as given in {{tab-dilithium-artifacts}}. All artifacts are encoded as
-defined in [Dilithium-Subm].
+ML-DSA has the parameterization with the corresponding artifact lengths in
+octets as given in {{tab-mldsa-artifacts}}. All artifacts are encoded as
+defined in [FIPS-204].
 
-{: title="Dilithium parameters and artifact lengths in octets" #tab-dilithium-artifacts}
-Algorithm ID reference | Dilithium instance | Public key | Secret key | Signature value
-----------------------:| ------------------ | -----------| ---------- | ---------------
-35, 37, 39             | Dilithium3         | 1952       | 4000       | 3293
-36, 38, 40             | Dilithium5         | 2592       | 4864       | 4595
+{: title="ML-DSA parameters and artifact lengths in octets" #tab-mldsa-artifacts}
+Algorithm ID reference | ML-DSA                  | Public key | Secret key | Signature value
+----------------------:| ----------------------- | -----------| ---------- | ---------------
+35, 37, 39             | ML&#x2011;DSA&#x2011;65 | 1952       | 4000       | 3293
+36, 38, 40             | ML&#x2011;DSA&#x2011;87 | 2592       | 4864       | 4595
```

Comment:
```suggestion
Algorithm ID reference | ML-DSA    | Public key | Secret key | Signature value
----------------------:| --------- | -----------| ---------- | ---------------
35, 37, 39             | ML-DSA-65 | 1952       | 4000       | 3293
36, 38, 40             | ML-DSA-87 | 2592       | 4864       | 4595
```

Reply:
@TJ-91 suggested to insert those hyphen characters in https://github.com/openpgp-pqc/draft-openpgp-pqc/pull/60#discussion_r1307226240 in order to avoid unwanted line breaks. Is it not breaking weirdly on your side?

Reply:
I opted for Aron's suggestion to avoid any kind of special character encodings. I think that ends up messy. Let it break as it breaks.